### PR TITLE
Translation

### LINKED ISSUE #64: Add translation features
* use attributes with `_fr` suffix
* directly translate some keywords (e.g. time in timeseries)

### PR DESCRIPTION
**Pull request by aulemahal**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
  - This PR fixes #64
- [x] (If applicable) Documentation has been added / updated (for bug fixes / features).
- [ ] (If applicable) Tests have been added.
- [ ] HISTORY.rst has been updated (with summary of main changes).
  - [ ] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added.

### What kind of change does this PR introduce?

* New function `get_localized_term` to get translated version of fixed terms. The translations are stored in the new `data/terms.yml` file.
* Modified `get_attributes` to automatically look for translated versions of the attributes it pulls.
* In both case, a `locale` arg can control which language to use, but the default behaviour is to take the first value in xclim's metadata_locales. 
* A minimal example is added to the notebook.

What do you think ?

### Does this PR introduce a breaking change?
No.

### Other information:
I also modified the `environment.yml` file with things that were present in `setup.py`. But I think there are too many dependencies listed here. I may make another PR for this.

### CODE REVIEW

**Comment by sarahclaude** [comment]
Should we add xclim in env.yml since it needs to be installed to use translate?

**Comment by aulemahal** [comment]
As you wish. xclim is already in the environment.yml file in this PR, but I do remove it in the other PR.

My reasoning was that xclim is not actually required to use Figanos. Not having xclim in an env when using figanos will never raise any error. Translation will even work if you use the `locale` args I added at some places. Real translation will only be activated through xclim's option, but I thought of that feature as "optional" and so xclim is also optional.

Also, I don't see why one would have figanos without xclim. But having fewer dependencies makes management easier, I think?

File: figanos/data/terms.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+time:
+  fr: temps
```
Comment (sarahclaude) [inline figanos/data/terms.yml:2]: ``    fr: Temps``

**Review by sarahclaude — APPROVED** [review]

File: figanos/data/terms.yml (hunk 1 above)
Comment (aulemahal) [reply figanos/data/terms.yml:2]: Hmm, je me rends compte qu'il y a une ambiguité avec les majuscules partout. Que penses-tu de : mettre les clés et les traductions en minuscules tout le temps et d'appeler ".capitalize()" là où nécessaire dans le code ?

Je le fais déjà à un endroit, mais pas partout.

Comment (sarahclaude) [reply figanos/data/terms.yml:2]: Donc, lorsque ce serait nécessaire on ajouterait capitalize et toutes les traductions de terms.yml seraient en minuscules? Ça me convient comme logique!

**Pull request merged**